include/beman/inplace_vector/inplace_vector.hpp

```diff
@@ -42,13 +43,42 @@ using inplace_vector_internal_size_type =
           If<Capacity <= std::numeric_limits<uint32_t>::max(), uint32_t,
              uint64_t>>>;
 
+// array based storage is used so that we can satisfy constexpr requirement
+//
+// Selecting this storage type implies: std::is_trivial_v<T>
+template <typename T, std::size_t Capacity>
+struct inplace_vector_array_based_storage {
+  std::array<T, Capacity> elems;
```

Comment:
Surprised initialization isn't needed here?

Reply:
> I think this is o.k. Why would we need initialisation before actually having any elements in the vector?

Does default constructor of std::array default construct all elements? I thought declaring uninitialized value is not allowed (including aggregate).

Reply:
Resolve this conversation as duplicate.

Reply:
suggestion implemented. `alignas(alignof(__T))` is not needed.